docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md

```diff
@@ -7,12 +7,73 @@ sidebar_position: 0
-automatically upgraded when Palette is upgraded. The three ways to activate this feature are:
+Palette supports the **Pause Agent Upgrades** feature to exclude a cluster or a group of clusters from having their
+Palette agent automatically upgraded when Palette is upgraded.
 
-- Pause Upgrades for a Single Cluster
-- Pause Upgrades for all Clusters within Project Scope
-- Pause Upgrades for all Clusters within Tenant Scope
+:::info
+
+This feature only pauses upgrades for Palette agents, not updates to the clusters themselves.
+
+:::
+
+## Pause Agent Upgrade Scopes
+
+Agent upgrades can be paused and resumed in the following scopes:
+
+- Pause agent upgrades for a single cluster
+- Pause agent upgrades for all clusters within a project
+- Pause agent upgrades for all clusters within a tenant
+
+When determining if the agent upgrades for one cluster is paused or not, you only need to look at the setting for the
+cluster itself. Even if agent upgrades are paused on a tenant or project level, agent upgrades for an individual cluster
+can still be turned on.
+
+Pausing or resuming agent upgrades at a higher-level scope will automatically pause or resume agent upgrades in the
+lower-level scopes. For example, if you pause agent upgrades at the tenant level, then agent upgrades will be paused for
+all projects within that tenant, and all clusters within those projects. Similarly, if you resume upgrades at the
+project level, then all clusters within that project will have their agent upgrades resumed.
+
+This is a one-time change that happens at the moment when you pause or resume upgrades in the project or tenant scope,
+and it does not mandate that the same setting be kept at the lower scopes. If you pause or resume agent upgrades in a
+lower-level scope, it will override the setting from the higher-level scope. For example, even if all agent upgrades are
+paused at the tenant level, you can override the tenant-level pause by resuming upgrades in a specific project or a
+specific cluster. However, if you resume upgrades at the tenant level, and then pause again at the tenant level, it will
+pause agent upgrades for all clusters within the tenant, including clusters where you manually overrode the tenant-level
+settings and resumed agent upgrades.
+
+The following table lists some example upgrade configurations and whether the Palette agent will be upgrades in those
+settings. Note that only the settings at the cluster level determines whether the Palette agent will be upgraded.
+
+| Tenant           | Project          | Cluster          | Outcome                                   |
+| ---------------- | ---------------- | ---------------- | ----------------------------------------- |
+| Upgrades paused  | Upgrades paused  | Upgrades enabled | Palette agent will upgrade automatically. |
+| Upgrades enabled | Upgrades enabled | Upgrade paused   | Palette agent upgrades are paused.        |
+
```

Comment:
Maybe I didn't understand this correctly. But if I override a cluster's settings. And I push out a new tenant or project setting. Will the new setting impact the cluster override setting I have?